src/@next/Link/Link.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import { Blue } from '../utilities/colors';
+import { StyledIcon, StyledLink, StyledTypography } from './LinkStyle';
+
+export interface LinkProps
+  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  children?: React.ReactNode;
+  id?: string;
+  url?: string;
+  external?: boolean;
+  monochrome?: boolean;
+  removeUnderline?: boolean;
+}
+
+export const Link = ({
+  children,
+  external = false,
+  monochrome = false,
+  removeUnderline = false,
+  url,
+  ...props
+}: LinkProps) => {
+  const [isHovered, setIsHovered] = useState(false);
```

Comment:
any reason we need to define the hover state?

Reply:
yes, I'm using it to define the color for the Icon: https://github.com/glints-dev/glints-aries/pull/761/files#diff-0e4ea2ddfa524b9a05d27f841fdf9909c75d4b8fd7feeafae440fe2bc899d8b4R33-R44

since we need to have them changed when it is hovered. do you have other ideas on a better way to handle it?

Reply:
I think we can do `&:hover svg { fill: '#color'}` since icon is the child element of the link

Reply:
ohhh that's better!

Reply:
addressed! please check the changes